public/static/docs/user-guide/analytics.md

```diff
@@ -25,9 +25,8 @@ User and event data have a 14 month retention period.
 
 DVC's analytics record the following information per event:
 
-- The DVC version, e.g. `0.22.0`
-- The operating system information, e.g. `linux`, `ubuntu`, `14.04`, etc
-- The underlying version control system, e.g. `git`
```

Comment:
why removing this?

Reply:
Oh. I think I thought this was outdated. We used to talk about "SCM" before, planning to support others like Mercurial, but now it's just and always Git.

Reply:
Maybe I should just rephrase to "Whether version control system (`git`) is being used" ?

Restored for now (without "e.g.")

Reply:
yep, but we probably still send it

also, can it be `none`?

Reply:
I don't know what gets sent for `--no-scm` projects, @efiop?

Reply:
(p.s. I'm asking #engineering)

Reply:
OK I'm editing all this slightly to be less specific with the analytic category values after chatting with engineering team and based on https://github.com/iterative/dvc/blob/ccca126749969cbcbf4f443166fbb72de2744073/dvc/utils/__init__.py. See 2b2bfca2